Arduino-Bling/RobotBling/animations.ino

```diff
@@ -443,3 +443,73 @@ void flash(unsigned long timeInterval, uint32_t color, uint8_t brightness)
   setAllPixelsOff();
   showPixels();
 }
+
+void rainbowCycle(unsigned long SpeedDelay) 
+{
+  byte *c;
+  uint16_t i, j;
+
+  for(j=0; j<256*5; j++) 
+  { // 5 cycles of all colors on wheel
+    for(i=0; i< NUM_LEDS; i++) 
+	{
+      c=Wheel(((i * 256 / NUM_LEDS) + j) & 255);
+      pixels.setPixelColor(i, *c, *(c+1), *(c+2));
+    }
+    showPixels();
+    delay(SpeedDelay);
+  }
+}
+
+// used by rainbowCycle and theaterChaseRainbow
+byte * Wheel(byte WheelPos) 
+{
+  static byte c[3];
+
+  if(WheelPos < 85) 
+  {
+   c[0]=WheelPos * 3;
+   c[1]=255 - WheelPos * 3;
+   c[2]=0;
+  } 
+  else if(WheelPos < 170) 
+  {
+   WheelPos -= 85;
+   c[0]=255 - WheelPos * 3;
+   c[1]=0;
+   c[2]=WheelPos * 3;
+  } 
+  else 
+  {
+   WheelPos -= 170;
+   c[0]=0;
+   c[1]=WheelPos * 3;
+   c[2]=255 - WheelPos * 3;
+  }
+
+  return c;
+}
+
+void theaterChaseRainbow(unsigned long SpeedDelay) {
+  byte *c;
+
+  for (int j=0; j < 256; j++) 
```

Comment:
also throughout this file (including outside of your pr) i see different uses of non-indentation spaces, ex. `int j = 0` or `int j=0`.